examples/step-63/doc/results.dox

```diff
@@ -1,2 +1,440 @@
+
+<table align="center" border="1">
+<tr>
+  <th></th>
+  <th></th>
+  <th colspan="1">$Q_1$</th>
+  <th colspan="7">Smoother (smoothing steps)</th>
+</tr>
+<tr>
+  <th></th>
+  <th></th>
+  <th></th>
+  <th colspan="3">Jacobi (6)</th>
+  <th></th>
+  <th colspan="3">Block Jacobi (3)</th>
+</tr>
+<tr>
+  <th></th>
+  <th></th>
+  <th></th>
+  <th colspan="3">Renumbering Strategy</th>
+  <th></th>
+  <th colspan="3">Renumbering Strategy</th>
+</tr>
+<tr>
+  <th>Cells</th>
+  <th></th>
+  <th>DoFs</th>
+  <th>Downstream</th>
+  <th>Random</th>
+  <th>Upstream</th>
+  <th></th>
+  <th>Downstream</th>
+  <th>Random</th>
+  <th>Upstream</th>
+</tr>
+<tr>
+  <th>32</th>
+  <th></th>
+  <th>48</th>
+  <td>3</th>
+  <td>3</th>
+  <td>3</th> 
+  <th></th>
+  <td>3</th>
+  <td>3</th>
+  <td>3</th> 
+</tr>
+<tr>
+  <th>128</th>
+  <th></th>
+  <th>160</th>
+  <td>6</th>
+  <td>6</th>
+  <td>6</th>
+  <th></th> 
+  <td>6</th>
+  <td>6</th>
+  <td>6</th> 
+</tr>
+<tr>
+  <th>512</th>
+  <th></th>
+  <th>576</th>
+  <td>11</th>
+  <td>11</th>
+  <td>11</th>
+  <th></th>
+  <td>9</th>
+  <td>9</th>
+  <td>9</th>  
+</tr>
+<tr>
+  <th>2048</th>
+  <th></th>
+  <th>2176</th>
+  <td>15</th>
+  <td>15</th>
+  <td>15</th> 
+  <th></th>
+  <td>13</th>
+  <td>13</th>
+  <td>13</th> 
+</tr>
+<tr>
+  <th>8192</th>
+  <th></th>
+  <th>8448</th>
+  <td>18</th>
+  <td>18</th>
+  <td>18</th> 
+  <th></th>
+  <td>15</th>
+  <td>15</th>
+  <td>15</th> 
+</tr>
+<tr>
+  <th>32768</th>
+  <th></th>
+  <th>33280</th>
+  <td>20</th>
+  <td>20</th>
+  <td>20</th> 
+  <th></th>
+  <td>16</th>
+  <td>16</th>
+  <td>16</th> 
+</tr>
+<tr>
+  <th>131072</th>
+  <th></th>
+  <th>132096</th>
+  <td>20</th>
+  <td>20</th>
+  <td>20</th> 
+  <th></th>
+  <td>16</th>
+  <td>16</th>
+  <td>16</th> 
+</tr>
+</table>
+
+On the other hand, for multiplicative smoothers, we can speed up convergence by renumbering the DoFs/cells in the advection direction, and similarly, we can slow down convergence if we do the renumbering in the opposite direction. This is because the multiplicative smoothers transfer information from DoF to DoF, and the solutions at any one DoF of an advection-dominate problem is highly dependent on the solution at DoFs upstream of the advection direction. 
+
+As we increase $\varepsilon$, however, the problem becomes more diffusion-dominated and the effectiveness of DoF/cell renumbering drops. On the other hand, in the extreme case of $\varepsilon=0$ (advection-only), we have a 1st-order PDE and multiplicative methods become effective solvers (Note: special care must be taken for the boundary conditions in this case).
```

Comment:
I probably would not call the transition to diffusion-dominated scenarios as the "effectiveness of DoF/cell renumbering drops", implying that the diffusion-dominated case is the odd one out. From a multigrid perspective, I would rather say that a uniform/isotropic (diffusive) propagation of information is standard and the directional flow of information (advection) for small epsilon is the "odd" one out. So I would phrase it rather such that the advantage of the directional information captured by multiplicative methods with the right numbering disappears once we go to a diffusion-dominated case, but it is still "effective" in the multigrid sense.